navigation3/navigation3-ui/build.gradle

```diff
@@ -79,7 +83,7 @@ androidXMultiplatform {
         androidMain {
             dependsOn(jvmMain)
             dependencies {
-                api(project(":activity:activity-compose"))
+                api("androidx.activity:activity-compose:1.7.0-rc01")
```

Comment:
Why it's old version but not stable version? Why not 1.10? There are alphas for 1.12.0 already